src/vecexpr.jl

```diff
@@ -66,6 +66,7 @@ const VECEXPR_FLAG_ISTREE = 0x01
 const VECEXPR_FLAG_ISCALL = 0x10
 const VECEXPR_META_LENGTH = 4
 
+@inline Base.isless(a::VecExpr, b::VecExpr) = Base.isless(a.data, b.data)
```

Comment:
Why is this needed?

Reply:
I did a short experiment with sorting. Ideally we could implement binary search over enodes in eclasses in ematch which would be asymptotically faster than iterating the array of enodes in large eclasses, we could leave it as VecExpr is a thin wrapper over numerical arrays